core/disputes.go

```diff
@@ -365,6 +368,9 @@ func (n *OpenBazaarNode) ProcessDisputeOpen(rc *pb.RicardianContract, peerID str
 			return err
 		}
 		if state == pb.OrderState_AWAITING_PAYMENT || state == pb.OrderState_AWAITING_FULFILLMENT || state == pb.OrderState_PARTIALLY_FULFILLED || state == pb.OrderState_PENDING {
+			if err := n.SendProcessingError(DisputerID, orderID, pb.Message_DISPUTE_OPEN, myContract); err != nil {
+				log.Errorf("failed sending ORDER_PROCESSING_FAILURE to peer (%s): %s", DisputerID, err)
+			}
```

Comment:
If there was a way to request a specific message from the buyer or seller then it can be used here.
When a node receives DISPUTE_OPEN and it finds itself in the above state, then it should simply request the appropriate message from the peer. ProcessingError message should be sent in case an order reaches an unrecoverable or unstable state on a node. The situation here can be remedied by simply requesting a message.

Reply:
That is sort of what the ProcessingError is meant to do. But instead of saying, only send XYZ message, send ALL of the messages that you have that I don't upto and including XYZ.